src/sidebar/services/groups.ts

```diff
@@ -478,4 +490,71 @@ export class GroupsService {
       userid: 'me',
     });
   }
+
+  /**
+   * Fetch members for a group from the API and load them into the store as the
+   * members of the focused group.
+   */
+  async loadFocusedGroupMembers(groupId: string): Promise<void> {
```

Comment:
The focused group ID doesn't need to be an argument to this method as we always want to load members for the group that is currently focused, and the service can get that from the store.

Reply:
Yeah, I know you suggested that in the previous review, and at some point the method was implemented that way.

However, not passing a group ID here means the side effect in the component does not need to depend on the focused group ID, and therefore, it does not re-run if focused group changes while mounted.

One solution would be to still add the focused group ID as a dependency, but it wouldn't be immediately obvious why.

I think there has to be a way to still make that work. Perhaps ensuring focusedGroupMembers value changes when another group is focused. Right now it could be `null` already, so it wouldn't change and therefore, wouldn't trigger a re-render.

Anyway, I'll try to think about that separately, as this PR is big enough already.